### PR TITLE
Add main toolbar stats

### PR DESCRIPTION
**Pull request by kingthorin**
- Updated various classes adding stat counters to button/menu actions.
- Added change note to one paros class.

Part of zaproxy/zaproxy#8375

### CODE REVIEW

**Comment by kingthorin** [comment]
QuickStart, reports, and hud will have to be done in other PRs

File: zap/src/main/java/org/parosproxy/paros/view/MainFrame.java [hunk 1]
```diff
@@ -377,6 +379,7 @@ private JButton getShowAllTabsButton() {
                         @Override
                         public void actionPerformed(ActionEvent e) {
                             View.getSingleton().showAllTabs();
+                            Stats.incCounter("stats.ui.toolbar.button.tabs.showall");
```
Comment (thc202) [inline zap/src/main/java/org/parosproxy/paros/view/MainFrame.java:382]: I'd suggest prefix with "maintoolbar", we have other toolbars that could also have stats in the future.

File: zap/src/main/java/org/zaproxy/zap/extension/brk/BreakPanelToolbarFactory.java [hunk 2]
```diff
@@ -640,6 +640,7 @@ public void actionPerformed(ActionEvent e) {
             setBreakAll(false);
             setBreakRequest(false);
             setBreakResponse(false);
+            Stats.incCounter("stats.ui.toolbar.button.break.continue");
```
Comment (thc202) [inline zap/src/main/java/org/zaproxy/zap/extension/brk/BreakPanelToolbarFactory.java:643]: These actions are also triggered in the Break tab, just saying we'll not know in which toolbar they are actually being used.

Comment (kingthorin) [reply zap/src/main/java/org/zaproxy/zap/extension/brk/BreakPanelToolbarFactory.java:643]: I left these as "toolbar" but changed all the others to "maintoolbar".

I tried to find a parent or ancestor method to use in a conditional but didn't have any success/

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**